blog/migrations/0002_comment.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+"""Initial migration. Autocreated.
```

Comment:
Normally you may ignore/exclude migrations and any auto-generated modules in pre-commit's config